prow/pod-utils/decorate/podspec.go

```diff
@@ -194,24 +195,32 @@ func decorate(spec *kube.PodSpec, pj *kube.ProwJob, rawEnv map[string]string) er
 		var cloneArgs []string
 
 		if cp := pj.Spec.DecorationConfig.CookiefileSecret; cp != "" {
+			// my-cookie/.gitcookies => find my-cookie secret at /secrets/cookiefile/.gitcookies
+			// my-cookie => find my-cookie secret at /secrets/cookiefile/my-cookie
```

Comment:
not exactly necessary for this PR, but in the config validation we might want validate that it is one of these three cases.

Reply:
What are the three cases? Empty, foo, foo/bar?

Reply:
Is the concern that I'm going to set my decoration config to `cookiefile_secret = "I!Love/Me$ome%ymbols#`?

I'm struggling to see what the risk here is?

Reply:
No real risk I think, it's just helpful to validate in presubmit to prevent people from trying to use values that cannot work.